student.rb

```diff
@@ -3,8 +3,8 @@
 class Student < Person
```

Comment:
- [ ] The Student class should have a method of associating with the Classroom class.

  ```
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```

Reply:
Fixed! 🔨

Reply:
This has been done well